tests/integration/test_credential_view.py

```diff
@@ -11,9 +11,11 @@ def test_list(self, client, logged_in_user, team, credential):
         assert response.status_code == 200
         parsed = response.json()
         assert len(parsed) == 3
-        assert UUID(parsed[0]["public_identifier"]) == credential.public_identifier
-        assert parsed[0]["name"] == "System user X"
-        parsed_team = parsed[0]["team"]
+        credential_teamx = [c for c in parsed if c["name"] == "System user X"]
```

Comment:
This test would sometimes pass, sometimes fail - the return order doesn't seem to be consistent